components/clp-package-utils/clp_package_utils/scripts/native/compress.py

```diff
@@ -128,9 +127,9 @@ def handle_job(sql_adapter: SQL_Adapter, clp_io_config: ClpIoConfig, no_progress
 
 
 def _generate_clp_io_config(
-    logs_to_compress: List[str], parsed_args: argparse.Namespace
+    logs_to_compress: List[str], parsed_args: argparse.Namespace, clp_config: CLPConfig
```

Comment:
This is minor, but we try to organize function parameters from stable -> dynamic, input -> output. So in this case, we'd usually put `clp_config` first since, compared to the other arguments, it's less likely to change per invocation of this method.